build.sbt

```diff
@@ -1432,6 +1434,7 @@ lazy val frgaalJavaCompilerSetting =
   customFrgaalJavaCompilerSettings(targetJavaVersion)
 
 def customFrgaalJavaCompilerSettings(targetJdk: String) = Seq(
+  resolvers += Resolver.mavenLocal,
```

Comment:
This enables usage of JARs from `~/.m2/repository`. To install them update GraalVM to: https://github.com/oracle/graal/pull/8266 in particular commit 853702c407c8fb31116decfefddef87a71203154 and do:
```
graal/sdk$ mx maven-install
graal/truffle$ mx maven-install
```
these two commands install all Truffle & SDK JARs into local repository. The `sbt` build can then find them.

Reply:
As of 253ded5 I am able to execute:
```
enso$ sbt 
sbt> project runtime
sbt:runtime> testOnly *ForeignMethodInvokeTest -- -z org.enso.interpreter.test.ForeignMethodInvokeTest.testParallelInteropWithJavaScript
```
and that is enough for testing https://github.com/oracle/graal/pull/8266